File: 백준/Silver/10814. 나이순 정렬/나이순 정렬.py
from sys import stdin

n = int(stdin.readline().rstrip())

members = list(stdin.readline().split() for _ in range(n))

members_sorted = sorted(members, key=lambda x: int(x[0]))

for i in range(n):
    print(members_sorted[i][0], members_sorted[i][1])

File: 백준/Silver/10814. 나이순 정렬/나이순 정렬.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

const input = [];
rl.on("line", (line) => {
  input.push(line.trim());
  if (input.length === Number(input[0]) + 1) rl.close();
}).on("close", () => {
  const arr = input.slice(1, input.length).map((val) => val.split(" "));

  arr.sort((a, b) => Number(a[0]) - Number(b[0]));

  console.log(arr.map((val) => val.join(" ")).join("\n"));

  process.exit();
});
